go-controller/pkg/ovn/ovn_test.go

```diff
@@ -260,6 +270,20 @@ func NewOvnController(ovnClient *util.OVNMasterClientset, wf *factory.WatchFacto
 	return dnc, err
 }
 
+func (o *FakeOVN) InitAndRunANPController() {
+	// We always call admin and baseline admin net pol tests without using fake address set factory setup
+	// So let's create a proper factory for testing
+	addressSetFactory := addressset.NewOvnAddressSetFactory(o.nbClient, config.IPv4Mode, config.IPv6Mode)
+	o.controller.addressSetFactory = addressSetFactory
```

Comment:
nice catch! its because my PR is older than https://github.com/ovn-org/ovn-kubernetes/pull/3525
we used to only use fake address-sets until that point so I had to do this hack, but I can remove this now..
this is what happens when something sits for over 6 months and undergoes numerous rebases and refactors...

Reply:
done.